subjects/devops/string_tokenizer_count/README.md

```diff
@@ -0,0 +1,42 @@
+## string_tokenizer_count
+
+### Instructions
+
+Create a file string_tokenizer_count.py that contains a function tokenizer_counter which takes in a string as a parameter and returns a dictionary of words and their count in the string.
+
+- The function should remove any punctuation from the string and convert it to lowercase before counting the words.
+
+- The function should return a dictionary of words and their count, sorted alphabetically by word.
+
+### Usage
+
+Here is an example of how to use the function:
```

Comment:
In other subjects, the below approach has been used to cover this part.

Present the python script (e.g. `tests.py`) content and then, reporting the result of the console as follows:
```consoles
$ python3 test.py
<expected output>
$
```

Reply:
Thanks, done.